draft-ietf-jsonpath-base.md

```diff
@@ -1238,7 +1238,7 @@ the comparison is between numeric values which satisfy the comparison.
 of the same values using the operator `==`, `<=`, `>=`, respectively.
 
 Note that `==` comparisons between a structured value and any value, including the same structured value, yield false.
-Also `!=` comparisons between a structured value and any value, including the same structured value, yield true.
+Consequently `!=` comparisons between a structured value and any value, including the same structured value, yield true.
```

Comment:
[SENT_START_CONJUNCTIVE_LINKING_ADVERB_COMMA] A comma may be missing after the conjunctive/linking adverb ‘Consequently’. -> (Consequently,)

(Thanks for picking this up)